common/utils/RsaKeyGenerator.js

```diff
@@ -0,0 +1,75 @@
+'use strict';
+
+const Promise = require('bluebird');
+const {
```

Comment:
Do you need the braces?

Reply:
This is destructuring of ES6- where instead of doing something like `require('child_process').spawn`, one can destructure the exposed properties into variables